sentry/src/main/java/io/sentry/event/Event.java

```diff
@@ -264,9 +264,15 @@ void setEnvironment(String environment) {
         this.environment = environment;
     }
 
+    //CHECKSTYLE.OFF: JavadocMethod
     public Map<String, Object> getExtra() {
+        if (extra == null) {
+            extra = new HashMap<>();
```

Comment:
If a developer doesn't set their ProGuard configuration correctly, their extras will be silently missing. To help developers, should we:
- Add a log statement here indicating this is happening so the developer, if they read logcat, can know what's wrong?
- Update the JavaDoc around extras to clue developers into why things might be missing, e.g. `SentryClient.addExtra` and `SentryClient.setExtra`?